pkg/api/schema/sql.go

```diff
@@ -41,27 +42,50 @@ func EncodeParams(params map[string]interface{}) ([]*NamedParam, error) {
 }
 
 func asSQLValue(v interface{}) (*SQLValue, error) {
-	if v == nil {
+	if v == nil || (reflect.ValueOf(v).Kind() == reflect.Ptr && reflect.ValueOf(v).IsNil()) {
```

Comment:
just wondering if handling the nil pointer case is needed..., if not nil, it may return into an error, isn't it?

Reply:
IMO a nil pointer can be considered nil...but the idea came from a third party application that Is sending nil pointer to SQL package ...